solana/programs/wormhole-governance/src/instructions/governance.rs

```diff
@@ -125,15 +124,32 @@ impl From<AccountMeta> for Acc {
     }
 }
 
+/// Processes a VAA [wormhole_anchor_sdk::wormhole::PostedVaa] sent to this program via a Guardian set.
+/// The VAA's payload contains an instruction and relevant Accounts that it requires. This program
+/// performs verification of the VAA's contents and then performs a Cross Program Invocation if all
+/// verification succeeds.
+/// NOTE: `missing_owner_check` is disabled here: The VAA instruction may contain
+/// placeholder accounts with Pubkeys set to hard-coded values [OWNER] and [PAYER].
+/// These keys are overwritten. Because they are placeholders, the `owner` field is not important.
+#[allow(unknown_lints)]
+#[allow(missing_owner_check)]
 pub fn governance<'info>(ctx: Context<'_, '_, '_, 'info, Governance<'info>>) -> Result<()> {
     let vaa_data = ctx.accounts.vaa.data();
 
     let mut instruction: Instruction = vaa_data.clone().into();
 
+    // Iterate over a copy of all accounts provided in the VAA payload.
```

Comment:
I can remove this comment if it's too much. It helped me to reason about the code but may be redundant for someone who is more familiar with Anchor.

Reply:
I like this comment here